addons/sms/wizard/send_sms.py

```diff
@@ -66,7 +66,7 @@ def default_get(self, fields):
         model = self.env[active_model]
 
         records = self._get_records(model)
-        if getattr(records, '_get_default_sms_recipients'):
+        if hasattr(records, '_get_default_sms_recipients'):
```

Comment:
why not

```python
try:
    partners = getattr(...)()
    phone_numbers = []
    no_phone_partners = []
except AttributeError:
   partners = []
for partner in partners:
```

Reply:
I don't know if it would work, but even

```python
partners = getattr(records, '_get_default_sms_recipients', list)()
```
might work. Never actually done that though, passing a type as a default.

Reply:
@gdgellatly, I would agree that `try`/`except` around `getattr()` is in line with the EAFP principle, however, in your example the `try`/`except` would mask any `AttributeError` raised by `_get_default_sms_recipients()`.

Morover, I've tried to follow the [PR guidelines](https://github.com/odoo/odoo/wiki/Contributing#making-pull-requests) and keep my changes minimal.